bundles/org.openhab.binding.tellstick/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,16 @@
 	<description>This is the binding for Telldus Tellstick Basic, Duo, .Net and Z-Net Lite.</description>
 	<connection>hybrid</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>usb</service-type>
+			<match-properties>
+				<match-property>
+					<name>chipId</name>
+					<regex>1781:0C31</regex>
```

Comment:
Does case matter or not as you also created https://github.com/openhab/org.openhab.binding.zwave/pull/1912?

Reply:
You are correct. Well spotted.